### PR TITLE
Vending Machine no gravity fix

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

THE vending machine shouldnt be able to fall over when there's no gravity

## Why It's Good For The Game

this is the most important fix in gaming hsitory

## Changelog

:cl:
fix: Vending machine doesn't fall over if its weightless
/:cl:

### CODE REVIEW

**Comment by blessedmulligan** [comment]
Nanotrasen vending machines have micro gravity generators as a loss prevention feature.

**Comment by SmoSmoSmoSmok** [comment]
> Nanotrasen vending machines have micro gravity generators as a loss prevention feature.

zany headcanon

File: code/modules/vending/_vending.dm [hunk 1]
```diff
@@ -532,7 +532,7 @@ GLOBAL_LIST_EMPTY(vending_products)
 
 ///Tilts ontop of the atom supplied, if crit is true some extra shit can happen. Returns TRUE if it dealt damage to something.
 /obj/machinery/vending/proc/tilt(atom/fatty, crit=FALSE)
-	if(QDELETED(src))
+	if(QDELETED(src) || !has_gravity(src))
```
Comment (Fikou) [inline code/modules/vending/_vending.dm:535]: you should probably put this in the melee vending machine fighting, as tilting also happens when a vending machine is thrown at someone

Comment (SmoSmoSmoSmok) [reply code/modules/vending/_vending.dm:535]: > you should probably put this in the melee vending machine fighting, as tilting also happens when a vending machine is thrown at someone

do you mean like when someone's baseball batted towards it? i tested that its handled here

**Pull request merged**